%% BioMed_Central_Tex_Template_v1.06
%%                                      %
%  ms00_bmc_article.tex            ver: 1.06 %
%                                       %

%%IMPORTANT: do not delete the first line of this template
%%It must be present to enable the BMC Submission system to
%%recognise this template!!

%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
%%                                     %%
%%  LaTeX template for BioMed Central  %%
%%     journal article submissions     %%
%%                                     %%
%%          <8 June 2012>              %%
%%                                     %%
%%                                     %%
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%

%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
%%                                                                 %%
%% For instructions on how to fill out this Tex template           %%
%% document please refer to Readme.html and the instructions for   %%
%% authors page on the biomed central website                      %%
%% https://www.biomedcentral.com/getpublished                      %%
%%                                                                 %%
%% Please do not use \input{...} to include other tex files.       %%
%% Submit your LaTeX manuscript as one .tex document.              %%
%%                                                                 %%
%% All additional figures and files should be attached             %%
%% separately and not embedded in the \TeX\ document itself.       %%
%%                                                                 %%
%% BioMed Central currently use the MikTex distribution of         %%
%% TeX for Windows) of TeX and LaTeX.  This is available from      %%
%% https://miktex.org/                                             %%
%%                                                                 %%
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%

%%% additional documentclass options:
%  [doublespacing]
%  [linenumbers]   - put the line numbers on margins

%%% loading packages, author definitions

%\documentclass[twocolumn]{bmcart}% uncomment this for twocolumn layout and comment line below
\documentclass[doublespacing,linenumbers]{bmcart}

% Set relative path to all figures
\newcommand*{\floatRelativePath}{../out/gwas417/pval_5e-08/r2_0.1/kb_1000/window_1000000/75_75}%

%%% Load packages
\usepackage{amsthm,amsmath}
%\RequirePackage[numbers]{natbib}
%\RequirePackage[authoryear]{natbib}% uncomment this for author-year bibliography
\RequirePackage{hyperref}
\usepackage[utf8]{inputenc} %unicode support
%\usepackage[applemac]{inputenc} %applemac support if unicode package fails
%\usepackage[latin1]{inputenc} %UNIX support if unicode package fails

\usepackage{csvsimple}

%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
%%                                             %%
%%  If you wish to display your graphics for   %%
%%  your own use using includegraphic or       %%
%%  includegraphics, then comment out the      %%
%%  following two lines of code.               %%
%%  NB: These line *must* be included when     %%
%%  submitting to BMC.                         %%
%%  All figure files must be submitted as      %%
%%  separate graphics through the BMC          %%
%%  submission process, not included in the    %%
%%  submitted article.                         %%
%%                                             %%
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%

\def\includegraphic{}
\def\includegraphics{}

%%% Put your definitions there:
\startlocaldefs
\endlocaldefs

%%% Begin ...
\begin{document}

%%% Start of article front matter
    \begin{frontmatter}

%        \begin{fmbox}
            \dochead{Research}

%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
%%                                          %%
%% Enter the title of your article here     %%
%%                                          %%
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%

            \title{Pleiotropy of expression quantitative trait loci increases with the number of regulated genes, active cell types and tissues and bound transcription factors}

%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
%%                                          %%
%% Enter the authors here                   %%
%%                                          %%
%% Specify information, if available,       %%
%% in the form:                             %%
%%   <key>={<id1>,<id2>}                    %%
%%   <key>=                                 %%
%% Comment or delete the keys which are     %%
%% not used. Repeat \author command as much %%
%% as required.                             %%
%%                                          %%
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%

            \author[
                addressref={aff1},                   % id's of addresses, e.g. {aff1,aff2}
                corref={aff1},                       % id of corresponding address, if any
% noteref={n1},                        % id's of article notes, if any
                email={aitor.gonzalez@univ-amu.fr}   % email address
            ]{\inits{A.G.}\fnm{Aitor} \snm{González}}
%\author[
%  addressref={aff1,aff2},
%  email={john.RS.Smith@cambridge.co.uk}
%]{\inits{J.R.S.}\fnm{John R.S.} \snm{Smith}}

%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
%%                                          %%
%% Enter the authors' addresses here        %%
%%                                          %%
%% Repeat \address commands as much as      %%
%% required.                                %%
%%                                          %%
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%

            \address[id=aff1]{%                           % unique id
%  \orgdiv{Department of Science},             % department, if any
                \orgname{Aix Marseille Univ, INSERM, TAGC},          % university, etc
                \city{Marseille},                              % city
                \cny{France}                                    % country
            }
%\address[id=aff2]{%
%  \orgdiv{Institute of Biology},
%  \orgname{National University of Sciences},
%  %\street{},
%  %\postcode{}
%  \city{Kiel},
%  \cny{Germany}
%}

%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
%%                                          %%
%% Enter short notes here                   %%
%%                                          %%
%% Short notes will be after addresses      %%
%% on first page.                           %%
%%                                          %%
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%

%\begin{artnotes}
%%\note{Sample of title note}     % note to the article
%\note[id=n1]{Equal contributor} % note, connected to author
%\end{artnotes}

%        \end{fmbox}% comment this for two column layout

%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
%%                                           %%
%% The Abstract begins here                  %%
%%                                           %%
%% Please refer to the Instructions for      %%
%% authors on https://www.biomedcentral.com/ %%
%% and include the section headings          %%
%% accordingly for your article type.        %%
%%                                           %%
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%

%        \begin{abstractbox}

            \begin{abstract} % abstract

                \parttitle{Background} %if any
                %
                Pleiotropic genetic variants affecting several traits are relatively common in the genome as shown by the increasing number of genome-wide association studies (GWAS).
%
                Complex disease variants are often located in non-coding regions and are likely gene regulatory variants.
%
                In this work, I investigate the link between frequent expression quantitative trait loci and pleiotropy.

                \parttitle{Results} %if any
% Verify numbers
                First I have computed the colocalization probability of 417 GWAS and 127 eQTL studies and I have observed 10,334 GWAS/eQTL pairs with at least one colocalized variants with a probability above 0.8.
%SELECT count(DISTINCT concat(`gwas_id`, `eqtl_id`)) FROM `gwas2eqtl_modelgwas2eqtl` where `PP_H4_abf`>=0.8; 10334
%Pourcentage of GWAS
%
                I have categorized the GWAS traits into 96 traits and categories to define pleiotropic as belonging at least to two of these categories.
%
                Pleiotropic variants are associated to more eQTL genes (eGenes) and more eQTL tissues (etissues) but egenes do also correlate with more traits.
%
                These observations could be partially explained, because pleitropic variants are enriched in splicing and missense variants and bind more transcription factors.
%
                These variants with the GWAS traits, egenes and etissues can be explored in a public website.

                \parttitle{Conclusions} %if any
                In conclusion, our work suggest that pleiotropy of gene regulatory variants arise from an increase of bound transcription factors, target egenes and etissues and severity of molecular effect consequences.
                %
            \end{abstract}

%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
%%                                          %%
%% The keywords begin here                  %%
%%                                          %%
%% Put each keyword in separate \kwd{}.     %%
%%                                          %%
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%

            \begin{keyword}
                \kwd{genetic variants}
                \kwd{expression quantitative trait loci}
                \kwd{complex diseases}
                \kwd{pleiotropy}
            \end{keyword}

% MSC classifications codes, if any
%\begin{keyword}[class=AMS]
%\kwd[Primary ]{}
%\kwd{}
%\kwd[; secondary ]{}
%\end{keyword}

%        \end{abstractbox}
%
%\end{fmbox}% uncomment this for two column layout

    \end{frontmatter}

%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
%%                                            %%
%% The Main Body begins here                  %%
%%                                            %%
%% Please refer to the instructions for       %%
%% authors on:                                %%
%% https://www.biomedcentral.com/getpublished %%
%% and include the section headings           %%
%% accordingly for your article type.         %%
%%                                            %%
%% See the Results and Discussion section     %%
%% for details on how to create sub-sections  %%
%%                                            %%
%% use \cite{...} to cite references          %%
%%  \cite{koon} and                           %%
%%  \cite{oreg,khar,zvai,xjon,schn,pond}      %%
%%                                            %%
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%

%%%%%%%%%%%%%%%%%%%%%%%%% start of article main body
% <put your article body there>

%%%%%%%%%%%%%%%%
%% Background %%

    \input{ms02_intro}
    \input{ms03_results}
    \input{ms04_discussion}
    \input{ms05_methods}
%  \input{ms06_floats}


    \section*{Appendix}
    Text for this section\ldots

%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
%%                                          %%
%% Backmatter begins here                   %%
%%                                          %%
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%

    \begin{backmatter}

        \section*{Acknowledgements}%% if any
        Centre de Calcul Intensif d'Aix-Marseille is acknowledged for granting access to its high performance computing resources.
%
        I thank Sandrine Marquet, Marie Michel, Pascale Paul, Salvatore Spicuglia and Pascal Rihet for helpful discussions and Léopoldine Lecerf for a preliminary version of the colocalization pipeline developed during her internship.

%    \section*{Funding}%% if any
%    Text for this section\ldots

%    \section*{Abbreviations}%% if any
%    Text for this section\ldots

        \section*{Availability of data and materials}%% if any
        Text for this section\ldots

%    \section*{Ethics approval and consent to participate}%% if any
%    Text for this section\ldots

        \section*{Competing interests}
        The authors declare that they have no competing interests.

%    \section*{Consent for publication}%% if any
%    Text for this section\ldots

        \section*{Authors' contributions}
        Text for this section \ldots

%    \section*{Authors' information}%% if any
%    Text for this section\ldots

%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
%%                  The Bibliography                       %%
%%                                                         %%
%%  Bmc_mathpys.bst  will be used to                       %%
%%  create a .BBL file for submission.                     %%
%%  After submission of the .TEX file,                     %%
%%  you will be prompted to submit your .BBL file.         %%
%%                                                         %%
%%                                                         %%
%%  Note that the displayed Bibliography will not          %%
%%  necessarily be rendered by Latex exactly as specified  %%
%%  in the online Instructions for Authors.                %%
%%                                                         %%
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%

% if your bibliography is in bibtex format, use those commands:
        \bibliographystyle{bmc-mathphys} % Style BST file (bmc-mathphys, vancouver, spbasic).
        \bibliography{ms_pleiotropy}      % Bibliography file (usually '*.bib' )
% for author-year bibliography (bmc-mathphys or spbasic)
% a) write to bib file (bmc-mathphys only)
% @settings{label, options="nameyear"}
% b) uncomment next line
%\nocite{label}

% or include bibliography directly:
% \begin{thebibliography}
% \bibitem{b1}
% \end{thebibliography}

%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
%%                               %%
%% Figures                       %%
%%                               %%
%% NB: this is for captions and  %%
%% Titles. All graphics must be  %%
%% submitted separately and NOT  %%
%% included in the Tex document  %%
%%                               %%
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%

%%
%% Do not use \listoffigures as most will included as separate files

        \section*{Figures}

        \begin{figure}[h!]
            \caption{Histogram showing the percentage of GWAS with the percentage of their tophits colocalized with eQTLs.
            I have excluded tophits in in the MHC locus in chromosome 6 between 25,000,000 and 35,000,000,
                because this locus is not included in the colocalization pipeline.}
            \label{fig:hist_perc_tophits_eqtl_excl_mhc}
        \end{figure}

%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
%%                               %%
%% Tables                        %%
%%                               %%
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%

%% Use of \listoftables is discouraged.
%%
        \section*{Tables}

        \input{ms07_tables}

%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
%%                               %%
%% Additional Files              %%
%%                               %%
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%

        \section*{Additional Files}
        \subsection*{Additional file 1: Table S1.}
        Classification of eQTL tissues and cell types. The first six were downloaded from the EBI eQTL repository.
        The 7th column "etissue\_category\_term" is used here to compute tissue diversity.

        \subsection*{Additional file 2: Table S2.}
        Metadata and classification of GWAS.

        \subsection*{Additional file 3: Table S3.}
        Percentage of tophits per GWAS that colocalized with at least one eQTL.

        \subsection*{Additional file 4: Table S4.}
        Count and list of GWAS phenotypes, egenes and etissues for each eQTL/GWAS variant.

        \subsection*{Additional file 5: Table S5.}
        Count and list of GWAS phenotypes, egene symbols and ENSEMBL IDs and etissue classes for each pleiotropic region.

    \end{backmatter}
\end{document}
