\leadauthor{Scientist}

\title{Phenotype pleiotropy of regulatory variants increases with the variety of bound transcription factors, regulated genes and active tissues}
\shorttitle{Running title here}

\author[1,\Letter]{Aitor Gonz√°lez \orcidlink{0000-0002-8402-1655}}
\author[2]{Second Doctor \orcidlink {000-0002-0000-0000}}
\author[1]{Third Professor \orcidlink {000-0003-0000-0000}}
\affil[1]{Aix Marseille Univ, INSERM, TAGC, 13288 Marseille, France}
\affil[2]{B Institute, Chalk Road, Blackboardville, USA}
\date{}

\maketitle

\begin{abstract}

Genome-wide association studies (GWAS) have shown that pleiotropic genetic variants affecting multiple traits are relatively common in the genome.
A large majority of these variants fall in non-coding regions and are likely gene regulatory variants.
Previous works investigated the pleitropy of frequent variants in the genome, but did not address specifically the link between gene regulation and pleiotropy.
%
In this work we have computed the colocalization of 413 GWAS studies and 125 eQTL studies and have observed XXX colocalized pairs with a probability above 0.8.
We have categorized the GWAS studies into XXX categories and have splitted variants according to the number of groups they belong to.
We observed that pleiotropic variants are associated to more eQTL genes (egenes), more eQTL tissues (etissues).
Unique egenes-etissues pairs are also associated to more GWAS phenotypes.
These observations could be partially explained, because pleitropic variants are enriched in splicing and missense variants and bind more transcription factors.
We provide a public ressource to explore these genome-wide eQTL/GWAS colocalizations.
%
Our work provide valuable information to better understand the molecular mechanism of pleitropy in the genome.


\lipsum[1][1]
\end{abstract}

\begin{keywords}
keyword1 | keyword2 | keyword3
\end{keywords}

\begin{corrauthor}
aitor.gonzalez\at univ-amu.fr
\end{corrauthor}

\section*{Introduction}\label{sec:introduction}

Genome-wide association studies (GWAS) evaluate the impact of frequent variants on a trait in the population.
%
The number of GWAS has rapidly increased since the first GWAS in 2010 and has result in comprehensive databases such as the NHGRI-EBI GWAS Catalog \citep{2018.Parkinson.Buniello}.
%
For instance, in 2019, NHGRI-EBI GWAS Catalog contained 5687 GWAS comprising 71673 variant-trait associations from 3567 publications \citep{2018.Parkinson.Buniello}.

However the molecular mechanism of GWAS variants is most often unclear because of three reasons \citep{2020.Trynka.CanoGamez}.
%
First, identification of the causaul variant is difficult because of linkage disequilibrium.
%
Second, GWAS do not provide hints regarding the tissues involved in the phenotypes.
%
Third, GWAS variants affect most often non-coding regions.

Expression QTLs (eQTLs) are variants that change the expression of a given gene in a tissue or cell type.
%
eQTLs have some of the same shortcommings as GWAS variants such as inability to identify the causal variant.
%
On the other hand, eQTLs provide information about the target genes and tissues.

eQTLs 


A large number of GWAS variants are found in non-conding regions.
To priori

Sentence describing EBI eQTL

In this work, I investigate the molecular basis of pleiotropy of gene regulatory variants.

\section*{Results}\label{s:results}

\subsection*{A pipeline to systematically colocalize eQTLs and GWAS variants}

I developed a pipeline to systematically colocalize eQTLs and GWAS variants.

I retrieved the better powered GWAS studies from the OpenGWAS database \citep{2018.Parkinson.Buniello}.

I selected physiological phenotypes that were well defined and excluded environmental traits.

This resulted in 413 GWAS studies and ??? loci (Supplementary table).

Then I manually organized these GWAS studies in categories to aggregate identical or similar phenotypes.

This annotation resulted in 96 categories that included between 1 and 24 GWAS identifiers (Supplementary table).

Then I retrieved all 125 eQTL studies from the EBI eQTL Catalogue at the date of XXX (Supplementary table).

This combination results potentially in 51,625 combinations.

%
125 eQTL studies were retrieved from the EBI eQTL Catalogue (Supplementary table).

Then, while GWAS variants were retrieved from the OpenGWAS database \citep{2018.Parkinson.Buniello,2021.Alasoo.Kerimov}.

 based on the EBI eQTL Catalogue and the NHGRI-EBI GWAS Catalog \citep{2018.Parkinson.Buniello,2021.Alasoo.Kerimov}.


\subsection*{Which regulatory variants are the most pleiotropic?}

We classified colocalized eQTL/GWAS variants according to the number of GWAS categories they belong to.

The list of variants involved in 5 or GWAS categories with the categories the variants are involved in are shown in table \ref{tab:pleitropic_variants}.

We can see that these 10 variants aggregate to 7 cytobands.

We can see the variants aggregate to pleiotropic regions, so that we also compute pleitropic region to explore them (See Methods).

\subsection*{How many pleiotropic regions are there}

We have found 454 regions with variants involved in 2 or more categories (Suppl. table region_window_100000.tsv).

Around 90\% regions, are shorter than 100 kb, 10\% are around 100 kb and a small fraction are larger than 100 kb.

The largest region is 11:13260511-17396930 (Centered in cytoband 11p15.2) with a length of 4,136,419 bp and variants involved in cardiovascular and hypertension phenotypes.
We find genes RRAS2, COPB1, CYP2R1 and SRY that are involved in mendelian diseases (REF TODO + OMIM).

The second largest region is 2:187235912-191066738 (Centered in cytoband 2q32.2) with a length of 3,830,826 bp and variants involved in cardiovascular, hypertension and autoimmune phenotypes (REF TODO + OMIM).
We find genes COL3A1, COL5A2 and SLC40A1.

\subsection*{Which genomic regions are the most pleiotropic?}

The list of pleiotropic regions (6 phenotype categories and more) is shown in Table \ref{tab:pleiotropic_regions}. The whole list will be given in a Supplementary file.

\subsection*{What is the function of the variants in the most pleiotropic regions?}

% 12q24.12
The cytoband 12q24.12 has five variants rs3184504, rs653178, rs7310615, rs597808, rs11065979, that are known in the literature to be associated to a wide palette of diseases.
This locus has been used to explain links between inflammaiton and hypertension (10.1097/MNH.0000000000000196), diabetes and autoimmunity (10.4239/wjd.v5.i3.316).
%
Our analysis add up that the variants in this cytoband control important genes such as ALDH2, ATXN2, MAPKAPK5, SH2B3 and TMEM116 in several tissues such as adipose, artery, blood, brain, digestive, skin, heart and skin tissue as well as immune cells.

% 5q31.1
The variant rs17622656 in locus 5q31.1 is involved in allergic, cardiovascular, hypertension and autoimmune diseases and is located in the intron of the IRF1 gene.
The IRF1 gene is a key activator gene for the innate and acquired immune responses.
In this locus there are rs736801 and rs2522051, which are upstream and downstream of IRF1 gene, respectively.

% 6p21.3
% TODO

%15q26.1
The next pleiotropic locus is in cytoband 15q26.1 involved in cardiovascular, hypertension and psyciatric diseases with variants rs2071382, rs8039305 and rs6224.
%
eQTLs in cytoband 15q26.1 control genes like Furin, oncogene Fes and UNC45A involved in Osteootohepatoenteric syndrome (OMIM DB).
%
eQTLs are active in a very large number of cells including blood, artery, brain and immune cells (See supplementary table).

% 1p13.3
The locus in cytoband 1p13.3 is mainly related to cardiovascular diseases.
Interestingly, these variants are active in many tissues (Supp table h4_annotated.ods).

The variant rs1344674 in cytoband 3q23 is involved in allergy, cancer and height phenotypes (tab 1 and count_per_rsid_gwas.tsv).
The target of this variant, which is an intron of the same gene, is ZBTB38 and it is active in the esophagus, blood and monocytes.

The variant rs2107595 is in locus 7p21.1 downstream of the histone deacetylase 9 (HDAC9) is cardiovascular. Interestingly, it controls expression of lncRNA XXX that is not known in the context of cardiovascular diseases.

The variant rs11236797 and the two others in locus 11q13.5 is mainly autoimmune.
All three regulate the gene EMSY-DT and are active induced pluripotent cells, LCDs and skin.

Another pleiotropic variants rs301806
rs301807
rs301802
rs301817
rs159963
rs301816
in the cytoband 1p36.23 that are involved in allergy, cardiovascular and hypertension.
This variants control the gene RERE, which is a nuclear receptor corregulator and is involved in neurodevelopmental disorders.\\

\subsection*{What is the frequency and proportion of variants with different phenotype categories, egene count and etissue categories?}

Next we wanted to explore the molecular basis of pleiotropy.

Figure \ref{fig:hist_gwas_egene_etissue}(a) shows that close to 99\% variants are involved in one GWAS category, 10\% are involved in 2 categories and less than 1\% of variants are involved in 5 GWAS categories

Figure \ref{fig:hist_gwas_egene_etissue}(b) shows that around 50\% are associated to 1-5 egenes, 1\% are associated to ...

Figure \ref{fig:hist_gwas_egene_etissue}(c) shows that around 50\% are active in 1-5 etissues, 1\% are associated to ...

%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
\subsection*{How do phenotype frequencies relates to eQTL and etissue frequency at the loci level?}

In Figure \ref{fig:region_gwas_egenes_tissues}, we can see the number of GWAS categories, eGenes and eTissues for regulatory variants in pleiotropic regions.

\subsection*{Where are the pleitropic variants?}

Based on the VEP consequence, we see that pleiotropic variants are increased in the coding regions (Missense) and 3' prime UTRs (Fig. \ref{fig:genome_location}).
By contrast pleiotropic variants decrease in introns.
This suggests their effects are stronger.

\subsection*{What is the mechanism of the pleiotropy? Are pleiotropic variants associated to more egenes or less genes?}

First we want to know if pleiotropic eQTLs are associated to more egenes.
The histograms of pleiotropic eQTLs are shifted to the righted compared to less pleiotropic eQTLs (Fig. \ref{fig:distrib_variant_egene_etissue_gwas}a).

\subsection*{What is the mechanism of the pleiotropy? Are pleiotropic egenes associated to more GWAS?}

Yes. There is a higher proportion of pleitropic egenes with higher number of GWAS traits (Fig. \ref{fig:distrib_variant_egene_etissue_gwas}b).

\subsection*{What is the mechanism of the pleiotropy? Are pleiotropic egenes associated to more etissues?}

Yes. There is a higher proportion of pleitropic egenes associated with higher number of etissues (Fig. \ref{fig:distrib_variant_egene_etissue_gwas}c).

\subsection*{How to explain that pleiotropic variants regulate more genes?}

In figure \ref{fig:freq_gwas_egene_etisue_per_variant}a, we have found that pleiotropic variants regulate more genes.
This observation could be explained by a non-significant tendency of pleiotropic variants to be in splicing and 3' UTR regions (Figure \ref{fig:vep_consequence}).

\subsection*{How to explain that pleiotropic egene affect more phenotypes?}

In figure \ref{fig:freq_gwas_egene_etisue_per_variant}c, we have found that egenes of pleiotropic variants regulate more phenotype.
This observation could be explained by a significant enrichment of pleiotropic variants to be missense variants (Figure \ref{fig:vep_consequence}a).

\subsection*{How tbo explain that pleiotropic variants are active in more tissues?}

In figure \ref{fig:freq_gwas_egene_etisue_per_variant}b, we have found that pleiotropic variants are active in more tissues.
To explain this observation, we hypothesize that genomic regions around pleiotropic variants bind more transcriptions factors.
In figure \ref{fig:freq_tf_per_variant}, we find that pleiotropic variants bind more transcription factors in a window with radius 50 nt around the variant.

\subsection*{What is the effect size of pleiotropic eQTLs?}

In figure \ref{fig:beta}, we find that pleiotropic variants have a smaller absolute effect size.

\subsection*{How can other people access these colocalized GWAS/eQTL?}

\begin{itemize}
  \item Remote and local access with Tabix
  \item UCSC browser track
  \item Downloadable SQLite database
  \item Downloadable raw TSV files
\end{itemize}

Text is added like this
This is a reference to a published paper \citep{2019.Watanabe}.
We can cite other things too \citep{watson_molecular_1953}

\section*{Methods}\label{s:methods}

\subsection*{Data sets and data exploration}

I used these data sets here: eQTLs from the eQTL Catalogue \citep{2021.Alasoo.Kerimov}, GWAS variants from the IEU OpenGWAS project \citep{2021.Marcora.Lyon}, chromatin immunoprecipitation (ChIP)	transcription factors peaks from the ReMap database \citep{2021.Ballester.Hammal}, UCSC annotation data.
%
I explore the data using the UCSC browser (\url{http://genome.ucsc.edu}) \citep{2021.Kent.Lee}, and OMIM database (\url{https://omim.org/}). 
%
Colocalization and analysis pipelines were implemented with Snakemake (Supplementary Figure S1) .
%
The colocalization and analysis pipelines can be found here: https://github.com/aitgon/eqtl2gwas


%
The developer documentation and scripts to download the data sets, anno-
tate the SNPs, train the model and score the genome can be
found here: https://github.com/aitgon/tagoos.


\subsection*{Colocalization}

The colocalization pipeline is managed by the Snakemake pipeline manager \citep{2021.Koester.Moelder}.

% Plagiat
Colocalization analyses were performed between 413 GWAS studies from the OpenGWAS databased and 125 eQTL studies from the EBI eQTL database.

The full list of GWAS with references is given in Supplementary Table xxx.

% Plagiat
To assess colocalization between GWAS loci and QTLs, I modified a pipeline from the github repository of the eQTL-Catalogue ( \url{https://github.com/eQTL-Catalogue/eQTL-Catalogue-resources} ).

First the lead eQTLs with FDR 0.05 are identified and their flanking region with radius 500 000 is tested.

\subsection*{Characterization of regulatory QTLs}


\subsection*{Definition of pleiotropic regions}



\section*{Bibliography}
\bibliographystyle{unsrt}
\bibliography{ms_pleiotropy.bib}


