\leadauthor{Scientist}

\title{Phenotype pleiotropy of regulatory variants increases with the variety of bound transcription factors, regulated genes and active tissues}
\shorttitle{Running title here}
	
\author[1,\Letter]{Aitor Gonz√°lez \orcidlink{0000-0002-8402-1655}}
\author[2]{Second Doctor \orcidlink {000-0002-0000-0000}}
\author[1]{Third Professor \orcidlink {000-0003-0000-0000}}
\affil[1]{Aix Marseille Univ, INSERM, TAGC, 13288 Marseille, France}
\affil[2]{B Institute, Chalk Road, Blackboardville, USA}
\date{}

\maketitle

\begin{abstract}

Genome-wide association studies (GWAS) have shown that pleiotropic genetic variants affecting multiple traits are relatively common in the genome.
A large majority of these variants fall in non-coding regions and are likely gene regulatory variants.
Previous works investigated the pleitropy of frequent variants in the genome, but did not address specifically the link between gene regulation and pleiotropy.
%
In this work we have computed the colocalization of 413 GWAS studies and 127 eQTL studies and have observed XXX colocalized pairs with a probability above 0.8.
We have categorized the GWAS studies into XXX categories and have splitted variants according to the number of groups they belong to.
We observed that pleiotropic variants are associated to more eQTL genes (egenes), more eQTL tissues (etissues).
Unique egenes-etissues pairs are also associated to more GWAS phenotypes.
These observations could be partially explained, because pleitropic variants are enriched in splicing and missense variants and bind more transcription factors.
We provide a public ressource to explore these genome-wide eQTL/GWAS colocalizations.
%
Our work provide valuable information to better understand the molecular mechanism of pleitropy in the genome.


\lipsum[1][1]
\end{abstract}

\begin{keywords}
keyword1 | keyword2 | keyword3
\end{keywords}

\begin{corrauthor}
aitor.gonzalez\at univ-amu.fr
\end{corrauthor}

\section*{Introduction}\label{sec:introduction}

Genome-wide association studies (GWAS) evaluate the impact of frequent variants on a trait in the population.
%
The number of GWAS has rapidly increased since the first GWAS in 2010 and has result in comprehensive databases such as the NHGRI-EBI GWAS Catalog \citep{2018.Parkinson.Buniello}.
%
For instance, in 2019, NHGRI-EBI GWAS Catalog contained 5687 GWAS comprising 71673 variant-trait associations from 3567 publications \citep{2018.Parkinson.Buniello}.

However the molecular mechanism of GWAS variants is most often unclear because of three reasons \citep{2020.Trynka.CanoGamez}.
%
First, identification of the causaul variant is difficult because of linkage disequilibrium.
%
Second, GWAS do not provide hints regarding the tissues involved in the phenotypes.
%
Third, GWAS variants affect most often non-coding regions.

Expression QTLs (eQTLs) are variants that change the expression of a given gene in a tissue or cell type.
%
eQTLs have some of the same shortcommings as GWAS variants such as inability to identify the causal variant.
%
On the other hand, eQTLs provide information about the target genes and tissues.

eQTLs 


A large number of GWAS variants are found in non-conding regions.
To priori

Sentence describing EBI eQTL

In this work, I investigate the molecular basis of pleiotropy of gene regulatory variants.

\section*{Results}\label{s:results}

\subsection*{A pipeline to systematically colocalize eQTLs and GWAS variants}

I developed a pipeline to systematically colocalize eQTLs and GWAS variants (Section \ref{sec:methods}).

125 eQTL studies were downloaded from the EBI eQTL catalogue, which aims to provide uniformly processed eQTLs in many tissues and cell types \citep{2021.Alasoo.Kerimov}.

Public GWAS summary statistics were selected from the IEU OpenGWAS database based on four criteria \citep{2018.Parkinson.Buniello}.
%
The first criterion was to exclude molecular data sets such as proteome or methylome.
%	
The second criterion was to include only the European population, because most samples from the EBI eQTL catalogue belong to the European population.
%
The third criterion was to keep only well-defined medical or physiological conditions and exclude environmental phenotypes such "employment status" or "self-reported" medical conditions.
%
The fourth criterion was to keep only GWAS studies with at least 10000 subjects, 2000 controls and 2000 cases (Supplementary table 2).
%
These filters resulted in 413 GWAS (Supplementary table 2).

I developed a colocalization pipeline that explored all the 51 625 combinations of eQTL studies and GWAS (Section Methods).
%
This analysis explored 30 261 unique variants and 2,479,065 potential variant colocalizations (OSF URL).
%
Selection of colocalizations with PP.H4.abf greater than 0.8 resulted in 9 758 variants and 143 119 colocalizations (Table TODO).

\subsection*{Which regulatory variants are the most pleiotropic?}

I manually assigned the 413 GWAS to 96 categories to aggregate identical or similar phenotypes (Supplementary table 2).
%
In the EBI eQTL Catalogue, immune cell types are annotated to great detail while other tissues are annotated at lower resolution.
%
Therefore I manually defined 36 categories to annotate eQTL biological samples at equivalent resolution (Supplementary table 1).

These category annotations were used to investigate the variant pleiotropy for GWAS phenotypes.
%
Colocalized eQTL/GWAS variants were classified according to the number of GWAS categories they belong to (Table \ref{tab:pleitropic_variants} and Supplementary table TODO).
%
Pleiotropic variants aggregate in some cytobands such as 3q23, 5q31.1, 11q13.5, 12q24.12 and 15q26.1 (Table \ref{tab:pleitropic_variants} and Supplementary table 3).

% 12q24.12
For instance, the most pleiotropic variants in cytoband 12q24.12 are involved in allergies, cancer, cardiovascular and autoimmune diseases (Table \ref{tab:pleitropic_variants}).
%
These variants control egenes ALDH2, LINC01405, MAPKAPK5, SH2B3 that are involved in alcohol-related disorders (ALDH2), cancer (LINC01405), multiple congenital anomalies syndromes (MAPKAPK5),  inflammation and hematological disorders (SH2B3).
%
These eQTLs are active in adipose tissue, arteries, blood, colon, immune cells, skin and induced pluripotent stem cells (iPSC).

%3q23
The variants in cytoband 3q23 are involved in allergy, cancer and height phenotypes (Supplementary table 3).
%
Interestingly all these variants target the egene ZBTB3, which is a transcription factor, which controls pro-inflammatory factors such as IRF5 (10.1016/j.jaut.2016.08.001).
%
These variants are active in tissues such as blood, esophagus, immune cell, LCLs and muscle (Supplementary table 3).

% 5q31.1
Another pleiotropic locus is cytoband 5q31.1 with variants that are involved in allergy, asthma, cardiovascular diseases, hypertension and ulcerative colitis.
%
eGenes under control of these variants are important genes of the immune system such as IRF1 and IL4.
%
Variant rs17622656 is in the intron of IRF1 gene and variants rs736801 and rs2522051, which are upstream and downstream of IRF1 gene, respectively.
%
IRF1 is a response protein to the presence of virus and oncogenic proteins.
%
IL4 is required to stimulate proliferation of activated B and T-cells.
%
There are other several other egenes in the region such as KIF3A, MIR3936HG, P4HA2, P4HA2-AS1, PDLIM4, RAD50, SEPTIN8, SLC22A4 and SLC22A5.
%
These variants are active in a large number of tissues including artery, blood, brain, breast, digestive and immune system, muscle, skin, ovary, testis and thyroid.

%11q13.5
The variants in cytoband 11q13.5 is involved in allergy and autoimmune disease (Supplementary table 3).
%
All these variants target the egene EMSY-DT, which is a long noncoding RNA upstream of EMSY.
%
These variants are active in LCL, Skin and iPSC (Supplementary table 3).

\subsection*{Analysis of pleiotropic regions}

As we saw, pleiotropic variants are aggregated in common cytobands.
%
Therefore I computed regions with a larger number of pleiotropic variants (See Methods) (Table \ref{tab:pleiotropic_regions}).
%
There are 453 regions with 441 regions with 2 or more categories, 114 regions with 3 or more categories, 37 region with 4 or more categories and 18 regions with 5 or more categories (Supplementary table 3).
%
75\% of regions are shorter than 100 kb, 15\% of regions are between 100 kb and 200 kb, and less than 5\% are larger than 200 kb (Fig. \ref{fig:pleiotropy_region_distribution}).

The most pleiotropic region is 12:111,395,984-111,645,358 in cytoband 12q24.12 that we discussed above (Table \ref{tab:pleiotropic_regions}).
%
The largest region is 11:13,260,511-17,396,930 in cytoband 11p15.2 with a length of 4,136,419 bp and variants involved in cardiovascular and hypertension phenotypes (Supplementary table 3).
%
The second largest region is 2:187,235,912-191,066,738 in cytoband 2q32.2 with a length of 3,830,826 bp and variants involved in cardiovascular, hypertension and autoimmune phenotypes  (Supplementary table 3).

\subsection*{What is the function of the variants in the most pleiotropic regions?}

% TODO GO DAVID

% 12q24.12
The cytoband 12q24.12 has five variants rs3184504, rs653178, rs7310615, rs597808, rs11065979, that are known in the literature to be associated to a wide palette of diseases.
This locus has been used to explain links between inflammaiton and hypertension (10.1097/MNH.0000000000000196), diabetes and autoimmunity (10.4239/wjd.v5.i3.316).
%
Our analysis add up that the variants in this cytoband control important genes such as ALDH2, ATXN2, MAPKAPK5, SH2B3 and TMEM116 in several tissues such as adipose, artery, blood, brain, digestive, skin, heart and skin tissue as well as immune cells.

% 6p21.3
% TODO

%15q26.1
The next pleiotropic locus is in cytoband 15q26.1 involved in cardiovascular, hypertension and psyciatric diseases with variants rs2071382, rs8039305 and rs6224.
%
eQTLs in cytoband 15q26.1 control genes like Furin, oncogene Fes and UNC45A involved in Osteootohepatoenteric syndrome (OMIM DB).
%
eQTLs are active in a very large number of cells including blood, artery, brain and immune cells (See supplementary table).

% 1p13.3
The locus in cytoband 1p13.3 is mainly related to cardiovascular diseases.
Interestingly, these variants are active in many tissues (Supp table h4_annotated.ods).



The variant rs2107595 is in locus 7p21.1 downstream of the histone deacetylase 9 (HDAC9) is cardiovascular. Interestingly, it controls expression of lncRNA XXX that is not known in the context of cardiovascular diseases.

The variant rs11236797 and the two others in locus 11q13.5 is mainly autoimmune.
All three regulate the gene EMSY-DT and are active induced pluripotent cells, LCDs and skin.

Another pleiotropic variants rs301806
rs301807
rs301802
rs301817
rs159963
rs301816
in the cytoband 1p36.23 that are involved in allergy, cardiovascular and hypertension.
This variants control the gene RERE, which is a nuclear receptor corregulator and is involved in neurodevelopmental disorders.\\

\subsection*{How specific or non-specific are variants to GWAS phenotypes, egenes and etissues?}

Next I explored the distribution of GWAS categories, egenes and etissues.
%
83\% of variants are involved in one GWAS category, 14\% are involved in 2 categories and the remaining variants with 3 or more phenotype categories make 3\% or less (Fig. \ref{fig:hist_gwas_egene_etissue}(a).
%
Regarding egenes, half of variants modulate one egene, and the other half modulate two or more egenes (Fig. \ref{fig:hist_gwas_egene_etissue}(b).
%
In the case of etissues, only 40\% of variants are specific to a single tissue while the other 60\% are active in two or more tissues (Fig. \ref{fig:hist_gwas_egene_etissue}(c).
%
In conclusion, while most colocalized eQTL/GWAS variants are specific to one GWAS phenotype, half of them only module one specific egene, and less than half are active in a single tissue.

%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
\subsection*{How do phenotype frequencies relates to eQTL and etissue frequency at the loci level?}

% TODO

In Figure \ref{fig:region_gwas_egenes_tissues}, we can see the number of GWAS categories, egenes and eTissues for regulatory variants in pleiotropic regions.

\subsection*{What are the mechanisms of pleiotropy}

\subsubsection*{More severe variant effect consequences?}

Then I wanted to know whether there is a significant difference of effect consequence between more and less pleiotropic variants.
%
I separated variants according to the count of GWAS categories and computed the EBI variant effect predictor (VEP) consequence.
%
I found a significant increase of the missense variants among those with 2, 3 and 4 categories compared to variants with 1 category (Fig. \ref{fig:vep_consequence}a).
%
There is also a significant increase of the splicing variants among those with 3 categories (Fig. \ref{fig:vep_consequence}b).
%
There is also a significant increase of the 3'-UTR variants among those with 4 categories (Fig. \ref{fig:vep_consequence}c).
%
This analysis suggest that more pleiotropic variants have a stronger effect on the coding sequence and splicing regions, which might explain partly their more pleiotropic function.

\subsubsection*{More egenes per variant?}

To find further explanations of pleiotropy, I computed whether pleiotropic variants have more egenes independently of the tissue.
%
I found that pleiotropic variants have significantly more egenes compared to variants with one GWAS category (Fig. \ref{fig:gwas_egene_etisue_per_variant}a).
%
This observation could be explained by my previous observation that pleiotropic variants have more often an effect on the splicing and 3'UTR regions (Figure \ref{fig:vep_consequence}b,c).

\subsubsection*{More etissues per variant-egene?}

The next hypothesis is that pleiotropic variant-egene pairs are active in more tissues compared with variant-egene pairs with one GWAS category and this increases their probability to affect more GWAS categories.
%
Indeed, we find that pleitropic variant-egenes associated are associated with a higher number of etissues (Fig. \ref{fig:gwas_egene_etisue_per_variant}b).

\subsubsection*{More GWAS per variant-egene-etissue?}

Another possibility is that variant-egene-etissue triples correlate with more GWAS categories.
%
Indeed we found a significant increase of GWAS categories for pleiotropic variant-egene-etissue triple (Fig. \ref{fig:gwas_egene_etisue_per_variant}c).
%
This observation could be explained by my previous observation that pleiotropic variants have more often a missense effect (Fig. \ref{fig:vep_consequence}a).

\subsubsection*{How to explain that pleiotropic variants are active in more tissues?}

I wondered why pleiotropic variants are active in more tissues as show in Fig. \ref{fig:gwas_egene_etisue_per_variant}b.
%
To explain this observation, I hypothesized that genomic regions around pleiotropic variants bind more transcriptions factors, which upregulate the pleiotropic regions in more tissues.
%
I looked at the number of different transcription factors per variant bound in a window of 100 bp.
%
I found a significant increase of diversity of transcription factors per variants for pleiotropic variants (Fig. \ref{fig:freq_tf_per_variant}a).

%TODO CRMs and variants

\subsection*{What is the effect size of pleiotropic eQTLs?}

Then I wondered, how does the effect size (beta) and significance (p-value) relate to pleiotropy.
%
I found a significant decrease of effect size with the excepction of the GWAS effect size of variants with 3 categories (Fig. \ref{fig:beta_pval}a).

Then I looked at the siginificance of pleiotropic eQTLs and GWAS variants.
%
I found a significant decrease of the eQTL significance (Fig. \ref{fig:beta_pval}c).
%
By contrast, the tendency of GWAS significance is unclear (Fig. \ref{fig:beta_pval}d).

In summary, it seems that pleiotropic eQTLs are weaker with a weaker effect size and less significant p-values.
%
The effect size of GWAS seems also to be smaller for pleiotropic variants but the effect is less clear for the significance.

\section*{Discussion}

\section*{Methods}\label{sec:methods}

\subsection*{Data sets and data exploration}

I used these data sets here: eQTLs from the eQTL Catalogue \citep{2021.Alasoo.Kerimov}, GWAS variants from the IEU OpenGWAS project \citep{2021.Marcora.Lyon}, chromatin immunoprecipitation (ChIP)	transcription factors peaks from the ReMap database \citep{2021.Ballester.Hammal}, UCSC annotation data.
%
I explore the data using the UCSC browser (\url{http://genome.ucsc.edu}) \citep{2021.Kent.Lee}, and OMIM database (\url{https://omim.org/}).
%
Colocalization and analysis pipelines were implemented with Snakemake (Supplementary Figure S1) .
%
The colocalization and analysis pipelines can be found here: \url{https://github.com/aitgon/eqtl2gwas} and \url{https://github.com/aitgon/eqtl2gwas_pleiotropy}.

\subsection*{Colocalization}

I developed a colocalization pipeline based on a pipeline that is public at the eQTL Catalogue Github repository ( \url{https://github.com/eQTL-Catalogue/colocalisation} ).
%
First lead eQTLs with FDR 0.05 in each sample are selected and surrounding eQTL and GWAS variants in a radius of 500 000 nt are retrieved.
%
Colocalization between eQTL and GWAS variants is tested using the "coloc.abf" function of CRAN coloc for each window, each eQTL sample and each GWAS.

\subsection*{Definition of pleiotropic regions}

To define pleiotropic regions, I included "pleiotropic" variants defining as having more than 2 categories as long as the distance between the two "pleiotropic" variants is less than 100 000 bp.
%
Then the number of GWAS categories in Supplementary table TODO is given by the number of different variant categories in the region.

\subsection*{Characterization of regulatory QTLs}


\subsection*{Definition of pleiotropic regions}

\section*{Code availability}

The eQTL/GWAS variant colocalization pipeline code is available at this repository: \url{https://github.com/aitgon/eqtl2gwas}.
%
The analysis pipeline of the eQTL/GWAS colocalization data and the source code of this manuscript is available at this repository: \url{https://github.com/aitgon/eqtl2gwas_pleiotropy}.

\section*{Data availability}

The raw colocalization data in a single file is available at this OSF site: \url{https://osf.io/hvmje}.
%
A website to access significant colocalization results (PP.H4.abf \geq 0.8) is available at this URL TODO.

\section*{References}
\bibliographystyle{unsrt}
\bibliography{ms_pleiotropy.bib}


