\section*{Introduction}\label{sec:introduction}

Genome-wide association studies (GWAS) allow to define the impact of genetic variants on a trait in the population.
%
The number of GWAS has rapidly increased since the first GWAS in 2010 and has result in comprehensive databases such as the NHGRI-EBI GWAS Catalog \citep{2018.Parkinson.Buniello}.
%
For instance, in 2019, NHGRI-EBI GWAS Catalog contained 5687 GWAS comprising 71673 variant-trait associations from 3567 publications \citep{2018.Parkinson.Buniello}.

However the molecular mechanism of GWAS variants is most often unclear because of three reasons \citep{2020.Trynka.CanoGamez}.
%
First, identification of the causaul variant is difficult because of linkage disequilibrium.
%
Second, GWAS do not provide hints regarding the tissues involved in the phenotypes.
%
Third, GWAS variants affect most often non-coding regions.

Expression QTLs (eQTLs) are variants that change the expression of a given gene in a tissue or cell type.
%
eQTLs share some of the same shortcommings as GWAS variants such as inability to identify the causal variant.
%
Also the distribution of eQTLs and GWAS variants in the genome have been suggested to be different because of negative selection.

Despite these previous disadvantages eQTLs are still important data sets to interprete GWAS variants, because they provide gene regulatory information about the eQTL genes (egenes) and tissues (etissues) (REF).
%
In addition colocalization anlaysis can pinpoint more precisely the causal variants that either data set independently (REF).

The large number of GWAS has allowed to discover that many genetic variants are involved in different traits.
%
TODO explain Watanabe.
%
Because GWAS variants affect mostly non-coding regions, it makes sense to investigate trait pleiotropy in terms of non-coding gene regulatory functions.

For these reasons, I have taken advantage of two well formatted eQTL and GWAS data sets to develop a global colocalization analysis that includes 127 eQTL studies and 413 GWAS.
%
I have used these colocalization results to investigate the gene regulatory properties of pleiotropic variants.
%
This global colocalization analysis is also useful to help interpret GWAS variants.
