\section*{Results}\label{s:results}

\subsection*{A pipeline to systematically colocalize eQTLs and GWAS variants}

To better understand the role of gene regulatory variants for variant pleiotropy, I developed a pipeline to systematically colocalize eQTLs and GWAS variants (Section Methods).
%
127 eQTL studies were downloaded from the EBI eQTL catalogue, which aims to provide uniformly processed eQTLs in many tissues and cell types \citep{2021.Alasoo.Kerimov}.

Public GWAS summary statistics were selected from the IEU OpenGWAS database based on four criteria \citep{2018.Parkinson.Buniello}.
%
The first criterion was to exclude molecular traits such as proteome or methylome.
%	
The second criterion was to include only the European population, because most samples from the EBI eQTL catalogue belong to the European population.
%
The third criterion was to keep only well-defined medical or physiological conditions and exclude environmental traits such "employment status" or "self-reported" medical conditions.
%
The fourth criterion was to keep only GWAS studies with at least 10,000 subjects, 2,000 controls and 2,000 cases (Supplementary table 2).
%
These filters resulted in 418 GWAS.
%
Among these studies, there were 10,627 clumped leading variants with a p-value below 5e-8 from 335 GWAS (Supplementary table 2).
% sqlite
% select count(distinct gwas_id) from tophits; 335
% select count(*) from (select DISTINCT chrom, pos, rsid, nea, ea from tophits) limit 50; 10627

I found eQTL colocalizations for 3,849 or 36$\%$ of the leading GWAS variants with 2,348 eQTL genes and 127 eQTL samples.
% sqlite
% select count(distinct tophits.rsid) from tophits inner join coloc on coloc.rsid=tophits.rsid where coloc.`PP.H4.abf`>=0.8; 3849
% alternative: select count(distinct tophits.rsid) from tophits, coloc where coloc.rsid=tophits.rsid and coloc.`PP.H4.abf`>=0.8; 3849
% select count(distinct coloc.egene) from tophits inner join coloc on coloc.rsid=tophits.rsid where coloc.`PP.H4.abf`>=0.8
% select count(distinct coloc.eqtl_id) from tophits inner join coloc on coloc.rsid=tophits.rsid where coloc.`PP.H4.abf`>=0.8

The largest classes of diseases by number of leading GWAS variants are autoimmune diseases (2107), cancer of breast (1976) and allergy (1321)
%
and the percentages of variants with a colocalized eQTL are 37\%, 21\% and 42\% in these classe, respectively (Supplementary table ST6).
%
%/* count tophits.rsid group by gwas_annot.gwas_class where coloc.`PP.H4.abf`>=0.8*/
%select gwas_annot.gwas_class, count(distinct tophits.rsid) from tophits, coloc, gwas_annot where gwas_annot.gwas_id=tophits.gwas_id and coloc.rsid=tophits.rsid and coloc.`PP.H4.abf`>=0.8 GROUP by gwas_annot.gwas_class;
%/* count tophits.rsid group by gwas_annot.gwas_class*/
%select gwas_annot.gwas_class, count(distinct tophits.rsid) from tophits, gwas_annot where gwas_annot.gwas_id=tophits.gwas_id GROUP by gwas_annot.gwas_class order by gwas_annot.gwas_class;
%
These results agree with previous work for autoimmune diseases \citep{2021.Li.Mu}.

This pipeline has resulted in 103,551 variants from 246 GWAS that colocalized (PP.H4.abf$\geq$0.8) with at least one eQTL.
%select count(*) from (select DISTINCT gwas_id FROM coloc where `PP.H4.abf`>=0.8);
%select count(*) from (select DISTINCT rsid FROM coloc where `PP.H4.abf`>=0.8);
%
I have developed a web application where I expose 116,736 variants from 246 GWAS that colocalize with at least one eQTL at a lower threshold value PP.H4.abf$\geq$0.5.
% select count(*) from (select DISTINCT rsid FROM coloc where `PP.H4.abf`>=0.5);
% select count(*) from (select DISTINCT gwas_id FROM coloc where `PP.H4.abf`>=0.5);

\subsection*{Pleiotropic variants colocalized with eQTLs}

\subsubsection*{Which variants are the most pleiotropic?}

I manually assigned the 418 GWAS to 96 categories to aggregate identical or similar traits (Supplementary table 1).
%
Then I verified whether GWAS traits cluster coherently within these classes.
%
I computed distances between GWAS traits based on the Spearman correlation of the colocalized eQTL beta of the different eQTL genes and samples.
%
I plot only classes with more than a minimum number of GWAS traits (Fig. \ref{fig:gwas_distance}).
%
GWAS traits cluster together in autoimmune diseases and circulatory system diseases.
%
Due to the large number of breast cancer GWAS, we have created a separeted class from cancer.
Nevertheless a weaker specific signal is observed between the breast cancer class and other types of cancers.

In the EBI eQTL Catalogue, immune cell types or neural tissues are annotated to great detail while other tissues are annotated at lower resolution.
%
Therefore I manually defined 36 categories to annotate eQTL samples at equivalent resolution (Supplementary table 2).

These category annotations were used to investigate the variant pleiotropy for GWAS traits.
%
Colocalized eQTL/GWAS variants were classified according to the number of GWAS categories they belong to (Table \ref{tab:pleitropic_variants} and Supplementary table 3).
%
These aggregated variants were highly pleiotropic.
%
For instance the variant rs2522051 is located upstream of the IRF1 gene in cytoband 5q31.1 and it is involved in six classes of traits, namely,
allergy, autoimmune diseases, cancer of breast, circulatory system diseases, hypertension and respiratory system diseases
(Table \ref{tab:pleitropic_variants} and Supplementary table 3).
%
This variant rs2522051 is active in 23 classes of cell type and tissues including adipose, brain, digestive cells, immune cells and sexual organs.
%
This variant rs2522051 regulates several genes such as IL13, PDLIM4 and RAD50.

\subsubsection*{What is the function of pleiotropic variants?}

%Then I evaluated the function of pleiotropic variants with 5, 4, 3 and 2 GWAS categories using gene ontology analysis.
%%
%I analyzed the ontology of eGenes with 5, 4, 3 and 2 GWAS categories using the DAVID web services \citep{2008.Lempicki.Huang,2008.Lempicki.Huang.NucleicAcidsResearch}.
%%
%I found that eGenes of pleiotropic variants with 3 and 2 GWAS categories were significantly enriched in functions of the immune system (Fig. \ref{s:results}{fig:geneontology}).
%For instance, eGenes in the cytoband 5q31.1 include IRF1 and IL4, which are important factors of the immune system.
%%
%IRF1 is a response protein to the presence of virus and oncogenic proteins and IL4 is required to stimulate proliferation of activated B and T-cells.
%%
%Variant rs17622656 is located in the intron of the IRF1 gene whereas variants rs736801 and rs2522051 are located upstream and downstream of the IRF1 gene, respectively.
%%
%These variants are associated with allergy, asthma, cardiovascular diseases, hypertension and ulcerative colitis and are active in a large number of tissues such as arteries, blood, brain, breast, digestive and immune system, muscle, skin, ovary, testis and thyroid.

%Variants of the 3q23 cytoband are involved in allergy, cancer and height traits (Supplementary table 3).
%%
%Interestingly all these variants target the eGene ZBTB3.
%%
%ZBTB3 is a transcription factor that controls pro-inflammatory factors such as IRF5 (10.1016/j.jaut.2016.08.001).
%%
%These variants are active in tissues such as blood, esophagus, immune cell, LCLs and muscle (Supplementary table 3).

%%11q13.5
%The variants in cytoband 11q13.5 are involved in allergy and autoimmune disease (Supplementary table 3).
%%
%All these variants target the eGene EMSY-DT, which is a long noncoding RNA upstream of EMSY.
%%
%These variants are active in LCL, Skin and iPSC (Supplementary table 3).
%
%In summary, I have identified pleiotropic variants with gene regulatory function.
%%
%Some of these loci are strongly involved in the immune system.

\subsection*{Pleiotropic regions}

Pleiotropic variants are aggregated in genomic regions.
%
Therefore I computed regions that include a large number of pleiotropic variants (See Methods) (Table \ref{tab:pleiotropic_regions}).
%
I found 13 regions with 5 or more GWAS classes, 30 regions with 4 or more GWAS classes and 80 regions with 3 or more GWAS.
%
75\% of regions are shorter than 100 kb, 15\% of regions are between 100 kb and 200 kb, and less than 5\% are larger than 200 kb (Fig. \ref{fig:pleiotropy_region_distribution}a).

The most pleiotropic region is 12:111,395,984-111,645,358 in cytoband 12q24.12 that we discussed above (Table \ref{tab:pleiotropic_regions}).
%
The largest region is 11:13,260,511-17,396,930 in cytoband 11p15.2 with a length of 4,136,419 bp and variants involved in cardiovascular and hypertension traits (Supplementary table 5).
%
The second largest region is 2:187,235,912-191,066,738 in cytoband 2q32.2 with a length of 3,830,826 bp and variants involved in cardiovascular, hypertension and autoimmune traits  (Supplementary table 5).

Very pleiotropic regions with 6 GWAS categories or more make less than 1Mb of the genome, while moderate pleiotropic regions with 3 GWAS categories or more make 2 Mb of the genome (Fig. \ref{fig:pleiotropy_region_distribution}b.

\subsection*{How specific are variants to GWAS traits, eGenes and eTissues?}

Next I explored the trait, eGene and eTissue specificity of eQTL/GWAS variants.
%
83\% of variants are involved in one GWAS category, 14\% are involved in 2 categories and the remaining variants with 3 or more trait categories make 3\% or less (Fig. \ref{fig:hist_gwas_egene_etissue}(a).
%
Regarding eGenes, half of variants modulate one eGene, and the other half modulate two or more eGenes (Fig. \ref{fig:hist_gwas_egene_etissue}(b).
%
In the case of eTissues, only 40\% of variants are specific to a single tissue while the other 60\% are active in two or more tissues (Fig. \ref{fig:hist_gwas_egene_etissue}(c).
%
In conclusion, most colocalized eQTL/GWAS variants are specific to one GWAS trait. By contrast, half of them only module one specific eGene, and less than half are active in a single tissue.

%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
\subsection*{How do trait frequencies relates to eQTL and eTissue frequency at the loci level?}

In Figure \ref{fig:region_gwas_egenes_tissues}, I have plotted the number of GWAS categories, eGenes and eTissues for regulatory variants in three pleiotropic regions.
%
All three regions are very pleiotropic with very different associated trait categories such as cancer, cardiovascular and autoimmune diseases.
%
However these plots show that the relationship between the number of traits, eGenes and eTissues can be very different.
%
The SLC22A5 locus in region 5:132,239,645-132,497,907 (5q31.1) has a low number of eGenes but a high number of eTissues.
%
By contrast, the MHC locus in region 6:31,034,839-32,478,149 (6p21.33) has a high number of both eGenes and eTissues.
%
Finally, the ATXN2 locus in region 12:111,395,984-111,645,358 (12q24.12) shows the highest trait pleiotropy but the number of eGenes and eTissues is very low.
%
In summary, trait pleiotropy can arise from different situations: few eGenes and eTissues, many eGenes and eTissues or a mix of both.

\subsection*{What are the mechanisms of pleiotropy}

Then, I wanted to understand the molecular mechanism of pleiotropic variants and regions.
%
I hypothesized that pleiotropy arises from bias in regulatory effects of variants.
%
For instance, pleiotropic variants might significantly affect some molecular functions more often.
%
Another possibility is that pleiotropic variants affect more eGenes in more eTissues, which affect more GWAS traits.

\subsubsection*{More severe variant effect consequences?}

I first analysed whether there are significant differences of some effect consequences between more and less pleiotropic variants.
%
I separated variants according to the count of GWAS categories and computed the EBI variant effect predictor (VEP) consequence \citep{2016.Cunningham.McLaren}.
%
I found a significant larger number of missense variants among variants with 2, 3 and 4 categories compared to variants with 1 category (Fig. \ref{fig:vep_consequence}a).
%
I also found a significant larger number of splicing variants among variants with 3 categories (Fig. \ref{fig:vep_consequence}b).
%
Finally, there is also a larger of the 3'-UTR variants among variants with 4 categories (Fig. \ref{fig:vep_consequence}c).
%
% TODO Discuss sQTLs in 2021.Li.Mu.GenomeBiology.impactcelltype paper
These analyses suggest that more pleiotropic variants have a stronger effect on the coding sequence and splicing regions, which might explain partly their more pleiotropic function.

\subsubsection*{More eGenes per variant-eTissue?}

Then I hypothesized that pleiotropic variants regulate more eGene even when we keep fixed the eTissue.
%
To test this hypothesis, the eGenes per variant-eTissues pairs were counted.
%
Then these eGene counts were classified according to the GWAS category count.
%
If the GWAS category count of a variant changed in different variant-eGene-eTissue trios, then we kept the maximal one.
%
I found means of eGene counts of 1.4, 1.7, 1.7 and 1.6 for GWAS categories counts 5, 4, 3 and 2 compared to eGene count mean of 1.5 for GWAS category count 1 (Fig. \ref{fig:gwas_egene_etisue_per_variant}a).
%
This suggests that pleiotropic variant-eTissue pairs have slightly but significantly (Mann–Whitney U test) more eGenes compared to variants with one GWAS category (Fig. \ref{fig:gwas_egene_etisue_per_variant}a).
%
This observation could be explained by my previous observation that pleiotropic variants have more often an effect on the splicing and 3'UTR regions (Figure \ref{fig:vep_consequence}b,c).

\subsubsection*{More eTissues per variant-eGene?}

My next hypothesis was that pleiotropic variant-eGene pairs are active in more eTissues compared with variant-eGene pairs with one GWAS category and this increases their probability to affect more GWAS categories.
%
To test this hypothesis, the eTissues per variant-eGene pairs were counted.
%
Then these eTissue counts were classified according to the GWAS category count.
%
If the GWAS category count of a variant changed in different variant-eGene-eTissue trios, then we kept the maximal GWAS category count.
%
I found means of eTissue counts of 2.2, 3, 3 and 2.6 for GWAS categories counts 5, 4, 3 and 2 compared to eTissue count mean of 2.5 for GWAS category count 1 (Fig. \ref{fig:gwas_egene_etisue_per_variant}b).
%
This suggests that pleiotropic variant-eGene pairs are active slightly but significantly (Mann–Whitney U test) in more eTissues compared to variants with one GWAS category (Fig. \ref{fig:gwas_egene_etisue_per_variant}b).

\subsubsection*{More GWAS per variant-eGene-eTissue?}

My next hypothesis was that pleiotropic variants are associated to more GWAS categories even after taking into account differences in eGene and eTissue counts.

To test this hypothesis, the GWAS category counts per variant-eGene-eTissue trios were counted.
%
Then these GWAS categorie counts per variant-eGene-eTissue trios were classified according to the GWAS category count per variant.
%
If the GWAS category count per variant changed in different variant-eGene-eTissue trios, then we kept the maximal GWAS category count per variant.
%
I found means of GWAS category counts per variant-eGene-eTissue of 3, 2, 1.6, 1.4 and 1 for GWAS categories counts per variant of 5, 4, 3, 2 and 1 (Fig. \ref{fig:gwas_egene_etisue_per_variant}c).
%
This shows a significant larger number of GWAS categories even in unique trios of variant-eGene-eTissue (Fig. \ref{fig:gwas_egene_etisue_per_variant}c).
%
This observation could be explained by my previous observation that pleiotropic variants have more often missense effects (Fig. \ref{fig:vep_consequence}a).

\subsubsection*{How to explain that pleiotropic variants are active in more tissues?}

In Fig. \ref{fig:gwas_egene_etisue_per_variant}b, I observed that pairs of eQTL-eGene are active in more eTissues.
%
To explain this observation, I hypothesized that pleiotropic variants are active in more eTissues, because genomic regions around pleiotropic variants bind more transcription factors, which upregulate the pleiotropic regions in more tissues.
%
To test this hypothesis, I counted the number of unique transcription factors bound in a radius of 50 bp around each variant (Window of 100 bp).
%
I found a significant larger number of transcription factors bound around pleiotropic variants (Fig. \ref{fig:freq_tf_per_variant}a).

Cis-regulatory modules (CRMs) are non-coding genomic regions with a higher density of bound cis-regulatory modules \citep{2021.Ballester.Hammal}.
%
I found that the odds ratio of variants annotated with CRMs vs non-annotated is significantly (Fisher's exact test) higher for variants with 2 and 3 GWAS category counts compared to category count 1 (Fig. \ref{fig:freq_tf_per_variant}b).

\subsection*{How do effect size and significance relate to pleiotropy}

Pleiotropic variants affect simultaneously more GWAS traits.
%
Therefore it is interesting to investigate how does variant causality and pleiotropy relate to effect size (beta) and significance (p-value) at the level of eQTL and GWAS traits.
%
I found that the mean of the absolute eQTL effect sizes (beta) decreased between GWAS category counts 1 and 5 (Fig. \ref{fig:beta_pval}a).
%
Regarding the eQTL significance (Negative decimal logarithm of the p-values), I found decreasing mean values for GWAS category counts between 1 and 5 (Fig. \ref{fig:beta_pval}b).

Then I carried out the same analysis for the GWAS effect size (beta) and significance (p-value).
%
I found that the mean of the absolute GWAS effect sizes (beta) decreased between GWAS category counts 1 and 5 (Fig. \ref{fig:beta_pval}b).
%
Regarding the GWAS variant significance (Negative decimal logarithm of the p-values), I found increasing mean values for GWAS category counts between 1 and 5 (Fig. \ref{fig:beta_pval}b).

These observations suggest that the strength of both eQTL and GWAS effects decrease with the pleiotropy (Fig. \ref{fig:beta_pval}a,b).
