\section*{Results}\label{s:results}

\subsection*{A pipeline to systematically colocalize eQTLs and GWAS variants}

To better understand the role of gene regulatory variants for variant pleiotropy, I developed a pipeline to systematically colocalize eQTLs and GWAS variants (Section Methods).
%
127 eQTL studies were downloaded from the EBI eQTL catalogue, which aims to provide uniformly processed eQTLs in many tissues and cell types \citep{2021.Alasoo.Kerimov}.

Public GWAS summary statistics were selected from the IEU OpenGWAS database based on four criteria \citep{2018.Parkinson.Buniello}.
%
The first criterion was to exclude molecular traits such as proteome or methylome.
%	
The second criterion was to include only the European population, because most samples from the EBI eQTL catalogue belong to the European population.
%
The third criterion was to keep only well-defined medical or physiological conditions and exclude environmental traits such "employment status" or "self-reported" medical conditions.
%
The fourth criterion was to keep only GWAS studies with at least 10,000 subjects, 2,000 controls and 2,000 cases (Supplementary table 2).
%
These filters resulted in 418 GWAS.
%
Among these studies, there were 10,627 clumped leading variants with a p-value below 5e-8 from 335 GWAS (Supplementary table 2).
% sqlite
% select count(distinct gwas_id) from tophits; 335
% select count(*) from (select DISTINCT chrom, pos, rsid, nea, ea from tophits) limit 50; 10627

My pipeline uncovered eQTL colocalizations for 3,849 or 36$\%$ of the leading GWAS variants with 2,348 eQTL genes and 127 eQTL samples.
% sqlite
% select count(distinct tophits.rsid) from tophits inner join coloc on coloc.rsid=tophits.rsid where coloc.`PP.H4.abf`>=0.8; 3849
% alternative: select count(distinct tophits.rsid) from tophits, coloc where coloc.rsid=tophits.rsid and coloc.`PP.H4.abf`>=0.8; 3849
% select count(distinct coloc.egene) from tophits inner join coloc on coloc.rsid=tophits.rsid where coloc.`PP.H4.abf`>=0.8
% select count(distinct coloc.eqtl_id) from tophits inner join coloc on coloc.rsid=tophits.rsid where coloc.`PP.H4.abf`>=0.8

The largest classes of diseases by number of leading GWAS variants are autoimmune diseases (2107), cancer of breast (1976) and allergy (1321)
%
and the percentages of variants with a colocalized eQTL are 37\%, 21\% and 42\% in these classe, respectively (Supplementary table ST6).
%
%/* count tophits.rsid group by gwas_annot.gwas_class where coloc.`PP.H4.abf`>=0.8*/
%select gwas_annot.gwas_class, count(distinct tophits.rsid) from tophits, coloc, gwas_annot where gwas_annot.gwas_id=tophits.gwas_id and coloc.rsid=tophits.rsid and coloc.`PP.H4.abf`>=0.8 GROUP by gwas_annot.gwas_class;
%/* count tophits.rsid group by gwas_annot.gwas_class*/
%select gwas_annot.gwas_class, count(distinct tophits.rsid) from tophits, gwas_annot where gwas_annot.gwas_id=tophits.gwas_id GROUP by gwas_annot.gwas_class order by gwas_annot.gwas_class;
%
These results agree with previous work for autoimmune diseases \citep{2021.Li.Mu}.

This pipeline resulted in 103,551 variants from 246 GWAS that colocalized (PP.H4.abf$\geq$0.8) with at least one eQTL.
%select count(*) from (select DISTINCT gwas_id FROM coloc where `PP.H4.abf`>=0.8);
%select count(*) from (select DISTINCT rsid FROM coloc where `PP.H4.abf`>=0.8);
%
I have also developed a web application where I expose 116,736 variants from 246 GWAS that colocalize with at least one eQTL at a lower threshold value PP.H4.abf$\geq$0.5.
% select count(*) from (select DISTINCT rsid FROM coloc where `PP.H4.abf`>=0.5);
% select count(*) from (select DISTINCT gwas_id FROM coloc where `PP.H4.abf`>=0.5);

%\subsection*{Pleiotropic variants colocalized with eQTLs}

\subsection*{Which eQTLs are the most pleiotropic?}

I manually assigned the 418 GWAS to 96 classes to aggregate identical or similar traits (Supplementary table 1).
%
Then I verified whether GWAS traits cluster coherently within these classes.
%
I computed distances between GWAS traits based on the Spearman correlation of the colocalized eQTL beta of the different eQTL genes and samples.
%
I plot only classes with more than a minimum number of GWAS traits (Fig. \ref{fig:gwas_distance}).
%
GWAS traits cluster together in autoimmune diseases and circulatory system diseases.
%
Due to the large number of breast cancer GWAS, we have created a separeted class from cancer.
Nevertheless a weaker specific signal is observed between the breast cancer class and other types of cancers.

In the EBI eQTL Catalogue, some samples have been analysed at the cell type level while other samples are analyzed at the tissue level.
%
For instance, there is much detail resolution regarding the immune cell types.
%
Therefore I manually defined 36 classes to aggregate eQTL samples to equivalent resolution (Supplementary table 2).
%
For instance the different immune cell types such as monocyte, T-cell, etc are aggregated to an ImmuneCell class (Supplementary table 2).

These class annotations were used to investigate the variant pleiotropy.
%
Colocalized eQTL/GWAS variants were classified according to the number of GWAS classes they belong to (Table \ref{tab:pleitropic_variants} and Supplementary table 3).
%
The aggregated variants are highly pleiotropic.
%
For instance the variant rs2522051 is located downstream of the IRF1 gene in cytoband 5q31.1 and it is involved in six classes of traits, namely,
allergy, autoimmune diseases, cancer of breast, circulatory system diseases, hypertension, and respiratory system diseases
(Table \ref{tab:pleitropic_variants} and Supplementary table 3).
%
This variant rs2522051 is active in 23 classes of cell types and tissues including adipose, brain, digestive cells, immune cells, and sexual organs.
%
This variant rs2522051 regulates several genes such as IL13, PDLIM4 and RAD50.

\subsection*{Which genomic regions contain the most pleiotropic eQTLs?}

Pleiotropic eQTLs are concentrated in genomic regions such as in cytobands 3q23,
5q31.1, 9p21.3, 15q24.1, and 19q13.33 for eQTLs involved on five or more classes (Supplementary table 3).
%
Therefore I computed regions that include at least one pleiotropic eQTL in
sliding windows of 100,000 nt (See Methods) (Table \ref{tab:pleiotropic_regions} and Supplementary table 4).
%
I found 13 regions with 5 or more trait classes, 30 regions with 4 or more trait
classes and 80 regions with 3 or more trait classes (Supplementary table 4).
%
50\% of regions with at least two trait classes are shorter than 100 kb, 20\% of regions are
between 100 kb and 200 kb, 10\% between 200 kb and 300 kb, and 20\% are larger than 300 kb (Fig. \ref{fig:pleiotropy_region_distribution}a).

The most pleiotropic region is 5:131,912,097-132,802,472 in cytoband 5q31.1 which contains the interferon response factor 1 (IRF1) and interleukins IL3, IL4, IL5 and IL13 (Table \ref{tab:pleiotropic_regions}).
%
Interferons and interleukins are very important factors for the immune system and anti-viral, inflammation and cancer responses.
%
The largest region is 7:2,712,518-7,254,268 in cytoband 7p22.3 with a length of 4,541,751 bp and variants associated to autoimmune and respiratory diseases and height (Supplementary table 5).

Pleiotropic regions with 3 or more trait classes make around 3 Mb of the genome (Fig. \ref{fig:pleiotropy_region_distribution}b).
Very pleiotropic regions with 5 trait classes or more make around 0.6 Mb of the genome,
regions with 4 GWAS classes make an additional 1 Mb and regions with 3 GWAS classes make an additional 1.5 Mb (Fig. \ref{fig:pleiotropy_region_distribution}b).

\subsection*{How specific are eQTLs regarding traits, genes and tissues?}

Next I verified whether eQTLs tend to be specific to traits, eQTL genes and eQTL tissues.
%
74\% of eQTLs are associated to one GWAS class, 21\% are associated to 2 classes and the remaining variants associated to 3 or more trait classes make less that 4\% of eQTLs (Fig. \ref{fig:hist_gwas_egene_etissue}a).
%
Regarding eQTL genes, 36\% of the colocalized eQTL/GWAS variant modulate a single gene, 24\% modulate two genes, and the remaining 40\% modulate three or more genes (Fig. \ref{fig:hist_gwas_egene_etissue}b).
%
Concerning tissues, 27\% colocalized eQTL/GWAS variants are active in a single tissue, 16\% in two tissues, 11\% in three tissues, and the remaining 46\% in four or more tissues (Fig. \ref{fig:hist_gwas_egene_etissue}c).
%
In conclusion, trait pleiotropy is rather rare, while, regulation of least two gene in at least two tissues is rather common (Fig. \ref{fig:hist_gwas_egene_etissue}).

\subsection*{Trait, eQTL gene and tissue frequency of individual eQTLs in pleiotropic loci}

In Figure \ref{fig:region_gwas_egenes_tissues}, I have plotted the number of trait classes, eQTL genes and tissues for regulatory variants in three pleiotropic regions.
%
All three regions are very pleiotropic with very different associated trait classes such as cancer, cardiovascular and autoimmune diseases.
%
However these plots show that the relationship between pleiotropy and the number of eQTL genes and tissues is not direct.

Some regions have high number of eQTL genes and intermediate number of tissues like the interleukin locus in region 5:131912097-132802472 (5q31.1),
intermediate number of eQTL gene and variable number of tissues like the 11:65488118-65916060 locus (11q13.1),
high number of both eQTL genes and tissues like the 12:55,721,801-56,171,027 (12q13.2) region
and relatively low number of eQTL genes and tissue like the 19:48,668,049-48,756,272 locus (19q13.33) (Fig. \ref{fig:region_gwas_egenes_tissues}).

In summary, trait pleiotropy can arise in locus with variable absolute numbers of genes and tissues.


\subsection*{Where are the pleiotropic variants?}

%\subsection*{What are the mechanisms of pleiotropy}

In the following sections, I wonder about the the molecular mechanism of pleiotropic eQTLs.
%
%I hypothesized that pleiotropy arises from bias in regulatory effects of variants.
%%
%For instance, pleiotropic variants might significantly affect some molecular functions more often.
%%
%Another possibility is that pleiotropic variants affect more eGenes in more eTissues, which affect more GWAS traits.

First, I analysed whether there are significant differences of some effect consequences between more and less pleiotropic variants.
%
I separated variants according to the count of GWAS classes and computed the EBI variant effect predictor (VEP) consequence (\citep{2016.Cunningham.McLaren}).
%
I found a significant larger number of upstream, 5'-UTR, downstream and 3'-UTR variants
with 2, 3 and 4 and more classes compared to variants with one class (Fig. \ref{fig:vep_consequence}).
%
I also found a significant larger number of intron, and non-coding transcript variants
with 3 and 4 and more classes (Fig. \ref{fig:vep_consequence}).
%
These analyses suggest that more pleiotropic variants have a stronger effect on
the coding sequence and splicing regions, which might explain partly their more pleiotropic function.

\subsection*{Do pleiotropic eQTLs modulate more genes?}

Then I hypothesized that pleiotropic variants modulate more genes even within the same tissues.
%
To test this hypothesis, I computed the number of eQTL genes per variant-tissue pair for different trait class counts.
%
%If the GWAS class count of a variant changed in different variant-eGene-eTissue trios, then we kept the maximal one.
%
eQTLs involved in 1, 2, 3, 4, and 5 and more GWAS classes modulate an average of 1.3, 1.5, 1.6, 1.9 and 1.8 genes, respectively (Fig. \ref{fig:gwas_egene_etisue_per_variant}a).
%I found means of eGene counts of 1.4, 1.7, 1.7 and 1.6 for GWAS classes counts 5, 4, 3 and 2 compared to eGene count mean of 1.5 for GWAS class count 1 (Fig. \ref{fig:gwas_egene_etisue_per_variant}a).
%
This suggests that pleiotropic eQTLs modulate more genes compared to eQTLs involved in only one GWAS class (Fig. \ref{fig:gwas_egene_etisue_per_variant}a).
%
More target genes of eQTLs provides an explanation to the observed pleiotropy.

\subsection*{Are pleiotropic eQTLs active in more tissues?}

My next hypothesis was that pleiotropic eQTL-gene pairs are active in more tissues than non-pleiotropic eQTLs.
%
To test this hypothesis, the tissues per eQTL-gene pair was counted for eQTL involved in different GWAS class counts.
%
eQTLs involved in 1, 2, 3, 4, and 5 and more GWAS classes are active in an average of 2.4, 2.5, 2.8, 3.4 and 4.3 tissues, respectively (Fig. \ref{fig:gwas_egene_etisue_per_variant}a).
%
This suggests that pleiotropic eQTL are active in more tissues (Mann–Whitney U test) compared to non-pleiotropic eQTLs (Fig. \ref{fig:gwas_egene_etisue_per_variant}b).

%\subsection*{Are pleiotropic eQTLs associated to more traits in unique gene-tissue pairs?}
%
%My next hypothesis was that pleiotropic variants are associated to more GWAS classes even after taking into account differences in eGene and eTissue counts.
%
%To test this hypothesis, the GWAS class counts per variant-eGene-eTissue trios were counted.
%%
%Then these GWAS categorie counts per variant-eGene-eTissue trios were classified according to the GWAS class count per variant.
%%
%If the GWAS class count per variant changed in different variant-eGene-eTissue trios, then we kept the maximal GWAS class count per variant.
%%
%I found means of GWAS class counts per variant-eGene-eTissue of 3, 2, 1.6, 1.4 and 1 for GWAS classes counts per variant of 5, 4, 3, 2 and 1 (Fig. \ref{fig:gwas_egene_etisue_per_variant}c).
%%
%This shows a significant larger number of GWAS classes even in unique trios of variant-eGene-eTissue (Fig. \ref{fig:gwas_egene_etisue_per_variant}c).
%%
%This observation could be explained by my previous observation that pleiotropic variants have more often missense effects (Fig. \ref{fig:vep_consequence}a).

\subsection*{Are pleiotropic eQTLs bound by more transcription factors?}

In Fig. \ref{fig:gwas_egene_etisue_per_variant}b, I observed that pairs of eQTL-gene are active in more tissues.
%
To explain this observation, I hypothesized that pleiotropic eQTLs are active in more tissues,
because genomic regions around pleiotropic variants bind more transcription factors,
which upregulate the pleiotropic regions in more tissues.
%
To test this hypothesis, I counted the number of unique transcription factors bound in a radius of 50 bp around each eQTL (Window of 100 bp).
%
eQTLs involved in 1, 2, 3 and 4 and more GWAS classes bind a mean number of 17,
19, 23 and 27 transcription factors, respectively (Fig. \ref{fig:gwas_egene_etisue_per_variant}a).
%I found a significant larger number of transcription factors bound around pleiotropic variants (Fig. \ref{fig:freq_tf_per_variant}a).

Cis-regulatory modules (CRMs) are non-coding genomic regions with a higher density of bound cis-regulatory modules \citep{2021.Ballester.Hammal}.
%
I found that the odds ratio of variants annotated with CRMs vs non-annotated is significantly (Fisher's exact test) higher for variants with 2 and 3 GWAS class counts compared to class count 1 (Fig. \ref{fig:freq_tf_per_variant}b).

In summary, pleitropic eQTLs bind more unique transcription factors and are more
likely to belong to cis-regulatory modules.

\subsection*{Do pleiotropic eQTLs have stronger effect sizes?}

Pleiotropic eQTLs affect simultaneously more trait classes.
%
Therefore, I wonder about the relationship between pleiotropy, effect size and significance of eQTLs.
%
I found that the mean of the absolute eQTL effect sizes (beta) decreased between GWAS class counts 1 and 5 (Fig. \ref{fig:beta_pval}a).
%
Regarding the eQTL significance (Negative decimal logarithm of the p-values), I found decreasing mean values for GWAS class counts between 1 and 5 (Fig. \ref{fig:beta_pval}b).

Then I carried out the same analysis for the GWAS effect size (beta) and significance (p-value).
%
I found that the mean of the absolute GWAS effect sizes (beta) decreased between GWAS class counts 1 and 5 (Fig. \ref{fig:beta_pval}b).
%
Regarding the GWAS variant significance (Negative decimal logarithm of the p-values), I found increasing mean values for GWAS class counts between 1 and 5 (Fig. \ref{fig:beta_pval}b).

These observations suggest that the strength of both eQTL and GWAS effects decrease with the pleiotropy (Fig. \ref{fig:beta_pval}a,b).
