\leadauthor{Scientist}

\title{Trait pleiotropy of regulatory variants increases with the variety of bound transcription factors, regulated genes and active tissues}
\shorttitle{Running title here}

\author[1,\Letter]{Aitor Gonz√°lez \orcidlink{0000-0002-8402-1655}}
\author[2]{Second Doctor \orcidlink {000-0002-0000-0000}}
\author[1]{Third Professor \orcidlink {000-0003-0000-0000}}
\affil[1]{Aix Marseille Univ, INSERM, TAGC, 13288 Marseille, France}
\affil[2]{B Institute, Chalk Road, Blackboardville, USA}
\date{}

\maketitle

\begin{abstract}
Genome-wide association studies (GWAS) have shown that pleiotropic genetic variants affecting multiple traits are relatively common in the genome.
A large majority of these variants fall in non-coding regions and are likely gene regulatory variants

We compute the colocalization of ? GWAS traits and ? eQTLs to pinpoint ? eQTLs associated to at least one GWAS trait.
We categorized GWAS traits and segregated eQTLs according to belonging to one or more categories.
Pleiotropy was defined as being associated to more than one category.
Pleiotropic variants clustered in regions with larger number of pleiotropic variants.
Pleiotropic variants were associated to more etissues and egenes.
Interestingly pleiotropic variants bound more transcription factors.
These colocalization data are available here: ?.

In summary, pleitropy of regulatory variants results from a higher number of bound transcription factors, regulated genes and active tissues.


\lipsum[1][1]
\end{abstract}

\begin{keywords}
keyword1 | keyword2 | keyword3
\end{keywords}

\begin{corrauthor}
aitor.gonzalez\at univ-amu.fr
\end{corrauthor}

\section*{Introduction}\label{sec:introduction}

\lipsum[2][1]

\section*{Results}\label{s:results}

\subsection*{Which regulatory variants are the most pleiotropic?}

The list of variants involved in 5 or GWAS categories are shown in table \ref{tab:pleitropic_variants}. 

The variant rs1344674 is the only one that is not known in the literature.
This suggest that our genome-wide approach is able to point at functional variants.

The complete list of variants in any category is in Supplementary Table TODO.
Among the following variants involved in 4 categories, we made a number of observations.

First we see that variants do not belong to independent regions and cluster together in small regions.
For instance, variants rs159963, rs301802, rs301806, rs301807, rs301816 and rs301817 are found in a small region of around 23 kb in chromosome 1 in the intron of gene RERE.
Moreover the GWAS categories are identical in these variants that are close together.
This observation made us analyzed pleiotropic regions below.

The second observation is that these different GWAS categories are actually related.
For instance, the categories of the variants in the gene RERE allergic rhinitis, asthma, cardiovascular and hypertension.

% full size table is table*
\begin{table*}[]
  \caption{Variants involved in 5 or more GWAS categories.}\label{tab:pleitropic_variants}
\centering
\scriptsize
\hline
\csvreader[separator=tab,
tabular=rrrcp{0.6\textwidth},
head,
table head=\bfseries Chrom. & \bfseries Pos. (hg38) & \bfseries Variant & \bfseries Cat. ct. & \bfseries GWAS Categories\\\hline,
]{fig/tab_rsid_most_pleiotropic.tsv}{}% use head of csv as column names
{\csvcoli\ & \csvcolii\ & \csvcoliii\ & \csvcoliv & \csvcolv}% specify your coloumns here
\hline
\end{table*}

\subsection*{What is the frequency and proportion of variants with different phenotype categories, egene count and etissue categories?}

TODO: New plot that shows percentage of variants in different GWAS cat...

\begin{figure*}[]
\centering
%
\begin{subfigure}[]{.32\textwidth}
\textbf{a}
\\
\includegraphics[width=\textwidth]{\floatRelativePath/plt_hist_gwas_etissue_egene.py/hist_gwas.png}
\end{subfigure}
%
\begin{subfigure}[]{.32\textwidth}
\textbf{b}
\\
\includegraphics[width=\textwidth]{\floatRelativePath/plt_hist_gwas_etissue_egene.py/hist_egene.png}
\end{subfigure}
%
\begin{subfigure}[]{.32\textwidth}
\textbf{c}
\\
\includegraphics[width=\textwidth]{\floatRelativePath/plt_hist_gwas_etissue_egene.py/hist_etissue.png}
\end{subfigure}
%
\caption{\textbf{Probability density of GWAS phenotypes, eGene or eQTL samples.} (\textbf{a}) TODO.} \label{fig:nucleus}
\end{figure*}

\subsection*{Do the number of phenotype categories affect the frequency of eGenes and eQTLs?}

Yes (Fig. \ref{fig:gwas_cat_vs_egene_and_sample}).

\begin{figure*}[!]
\centering
%
\begin{subfigure}[]{.33\textwidth}
\textbf{a}
\\
\includegraphics[width=\textwidth]{\floatRelativePath/plt_vlnplt_gwas_egene_etissue.py/violinplot_gwas_egene.png}
\end{subfigure}
%
\begin{subfigure}[]{.33\textwidth}
\textbf{b}
\\
\includegraphics[width=\textwidth]{\floatRelativePath/plt_vlnplt_gwas_egene_etissue.py/violinplot_gwas_etissue.png}
\end{subfigure}
%
\caption{\textbf{Relation between GWAS category number and eGene and eQTL sample number.} (\textbf{a}) TODO.} \label{fig:gwas_cat_vs_egene_and_sample}
%
\end{figure*}

\subsection*{Where are the pleitropic variants?}

Based on the VEP consequence, we see that pleiotropic variants are increased in the coding regions (Missense) and 3' prime UTRs (Fig. \ref{fig:genome_location}).
By contrast pleiotropic variants decrease in introns.
This suggests their effects are stronger.

\begin{figure*}[!]
\centering
%
\begin{subfigure}[]{.33\textwidth}
\textbf{a}
\\
\includegraphics[width=\textwidth]{\floatRelativePath/plt_vep_consequence.py/missense_variant.png}
%
\end{subfigure}
%
\begin{subfigure}[]{.33\textwidth}
\textbf{b}
\\
\includegraphics[width=\textwidth]{\floatRelativePath/plt_vep_consequence.py/splice_region_variant.png}
%
\end{subfigure}
%
\begin{subfigure}[]{.33\textwidth}
\textbf{c}
\\
\includegraphics[width=\textwidth]{\floatRelativePath/plt_vep_consequence.py/3_prime_UTR_variant.png}
%
\end{subfigure}
%
\caption{\textbf{Location of of variants computed as VEP consequence.} (\textbf{a}) TODO.} \label{fig:gwas_cat_vs_egene_and_sample}
%
\end{figure*}

%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
\subsection*{How do phenotype frequencies relate pleiotropy relates to eQTLs and etissues at the loci level?}

In Figure \ref{fig:region_gwas_egenes_tissues}, we can see the number of GWAS categories, eGenes and eTissues for regulatory variants in pleiotropic regions.

\begin{figure*}[!ht]

%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
\begin{subfigure}[]{.33\textwidth}
\textbf{a}
\\
\includegraphics[width=\textwidth]{\floatRelativePath/plt_scttr_count_per_rsid_gwas.py/count_per_rsid_chr5_start132239645_end132497907_categories8.png}
\end{subfigure}
%
\begin{subfigure}[]{.33\textwidth}
\textbf{b}
\\
\includegraphics[width=\textwidth]{\floatRelativePath/plt_scttr_count_per_rsid_gwas.py/count_per_rsid_chr6_start31034839_end32478149_categories8.png}
\end{subfigure}
%
\begin{subfigure}[]{.33\textwidth}
\textbf{c}
\\
\includegraphics[width=\textwidth]{\floatRelativePath/plt_scttr_count_per_rsid_gwas.py/count_per_rsid_chr12_start111395984_end111645358_categories11.png}
\end{subfigure}

%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
\begin{subfigure}[]{.33\textwidth}
\textbf{d}
\\
\includegraphics[width=\textwidth]{\floatRelativePath/plt_scttr_count_per_rsid_egene.py/count_per_rsid_chr5_start132239645_end132497907_categories8.png}
\end{subfigure}
%
\begin{subfigure}[]{.33\textwidth}
\textbf{e}
\\
\includegraphics[width=\textwidth]{\floatRelativePath/plt_scttr_count_per_rsid_egene.py/count_per_rsid_chr6_start31034839_end32478149_categories8.png}
\end{subfigure}
%
\begin{subfigure}[]{.33\textwidth}
\textbf{f}
\\
\includegraphics[width=\textwidth]{\floatRelativePath/plt_scttr_count_per_rsid_egene.py/count_per_rsid_chr12_start111395984_end111645358_categories11.png}
\end{subfigure}

%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
\begin{subfigure}[]{.33\textwidth}
\textbf{g}
\\
\includegraphics[width=\textwidth]{\floatRelativePath/plt_scttr_count_per_rsid_etissue.py/count_per_rsid_chr5_start132239645_end132497907_categories8.png}
\end{subfigure}
%
\begin{subfigure}[]{.33\textwidth}
\textbf{h}
\\
\includegraphics[width=\textwidth]{\floatRelativePath/plt_scttr_count_per_rsid_etissue.py/count_per_rsid_chr6_start31034839_end32478149_categories8.png}
\end{subfigure}
%
\begin{subfigure}[]{.33\textwidth}
\textbf{i}
\\
\includegraphics[width=\textwidth]{\floatRelativePath/plt_scttr_count_per_rsid_etissue.py/count_per_rsid_chr12_start111395984_end111645358_categories11.png}
\end{subfigure}

\caption{\textbf{Count of GWAS categories, eGenes and eTissues in pleiotropic genomic regions.} (\textbf{a,d,g}) Region 5:132,239,645-132,497,907, \textbf{b,e,h}) region 6:31,034,839-32,478,149,  and \textbf{c,g,i}) region 12:111,395,984-111,645,358} \label{fig:region_gwas_egenes_tissues}
%
\end{figure*}

\subsection*{What is the mechanism of the pleiotropy? ? Are pleiotropic variants associated to more egenes or less genes?}

First we want to know if pleiotropic eQTLs are associated to more egenes.
The histograms of pleiotropic eQTLs are shifted to the righted compared to less pleiotropic eQTLs (Fig. \ref{fig:distrib_variant_egene_etissue_gwas}a).


\begin{figure*}[!]
\centering
%
\begin{subfigure}[]{.33\textwidth}
\textbf{a}
\\
\includegraphics[width=\textwidth]{\floatRelativePath/plt_violin_egene_per_variant.py/vlnplt.png}
\end{subfigure}
%
\begin{subfigure}[]{.33\textwidth}
\textbf{b}
\\
\includegraphics[width=\textwidth]{\floatRelativePath/plt_violin_etissue_per_variant_egene.py/vlnplt.png}
\end{subfigure}
%
\begin{subfigure}[]{.33\textwidth}
\textbf{c}
\\
\includegraphics[width=\textwidth]{\floatRelativePath/plt_violin_gwas_per_variant_egene_etissue.py/vlnplt.png}
\end{subfigure}
%
\caption{\textbf{TODO.} (\textbf{a}) TODO.} \label{fig:distrib_variant_egene_etissue_gwas}
%
\end{figure*}

\subsection*{What is the mechanism of the pleiotropy? ? Are pleiotropic egenes associated to more GWAS?}

Yes. There is a higher proportion of pleitropic egenes with higher number of GWAS traits (Fig. \ref{fig:distrib_variant_egene_etissue_gwas}b).

\subsection*{What is the mechanism of the pleiotropy? Are pleiotropic egenes associated to more etissues?}

Yes. There is a higher proportion of pleitropic egenes associated with higher number of etissues (Fig. \ref{fig:distrib_variant_egene_etissue_gwas}c).

\subsection*{How many pleiotropic regions are there}

We have found around 350 regions with variants involved in 2 or more categories (Fig. \ref{fig:pleiotropy_region_distribution}).
Around 250 are 100,000 bp in length, 100 regions are 200,000 bp and the others are longer.

\subsection*{Which genomic regions are the most pleiotropic?}

The list of pleiotropic regions (5 and 4 trait categories) is shown in Table \ref{tab:pleiotropic_regions}. The whole list will be given in a Supplementary file.

% full size table is table*
\begin{table*}[]
\caption{Pleiotropic regions involving more than 4 GWAS categories.}\label{tab:pleiotropic_regions}
\centering
\scriptsize
\hline
\csvreader[
separator=tab,
tabular=rrrcp{0.5\textwidth},
head,
table head=\bfseries Chrom. & \bfseries Start (hg38) & \bfseries End (hg38) & \bfseries Cat. ct. & \bfseries GWAS Categories\\\hline,
]{fig/tab_region_window_100000_pleio_highest.tsv}{}% use head of csv as column names
{\csvcoli\ & \csvcolii\ & \csvcoliii\ & \csvcoliv & \csvcolv}% specify your coloumns here
\hline
\end{table*}

\begin{figure*}[]
\centering
%
\includegraphics[width=0.33\textwidth]{\floatRelativePath/cmpt_pleiotropic_regions.py/regions_100000_length_hist.png}
%
\caption{\textbf{Length distribution of pleiotropic regions.} (\textbf{a}) TODO.} \label{fig:pleiotropy_region_distribution}
\end{figure*}

\subsection*{How to explain that pleiotropic variants are actives in more tissues?}

In figure xx, we have found that pleiotropic variants are active in more tissues.
To explain this observatio, we hypothesize that genomic regions around pleiotropic variants bind more transcriptions factors.
Indeed we find a bias towards more transcription factors around pleiotropic variants.

\begin{figure*}[!]
\centering
%
\begin{subfigure}[]{.33\textwidth}
%
\includegraphics[width=\textwidth]{\floatRelativePath/plt_bxplt_remaptf_per_rsid.py/bxplt_remaptf_per_rsid_flank_50.png}
\end{subfigure}
%
\caption{\textbf{Binding of transcription factors in the region (100 kb) around pleiotropic variants.} (\textbf{a}) TODO.} \label{fig:gwas_cat_vs_egene_and_sample}
%
\end{figure*}

\subsection*{How can other people access these colocalized GWAS/eQTL?}

\begin{itemize}
  \item Remote and local access with Tabix
  \item UCSC browser track
  \item Downloadable SQLite database
  \item Downloadable raw TSV files
\end{itemize}

\section*{Bibliography}
\bibliographystyle{bxv_abbrvnat}
\bibliography{refs.bib}
